db/migrations/2023_09_18_000000_permissions_rename_news_important_to_news_highlight.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Imho those migrations should be combined into one to make migrating easier

Reply:
what should be the name of the migration?